test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/HelpCommand.cs

```diff
@@ -9,8 +9,17 @@ public sealed class HelpCommand : DotnetCommand
     {
         public override CommandResult Execute(string args = "")
         {
-            args = $"help {args}";
-            return base.Execute(args);
+            return base.Execute(AppendHelp(args));
+        }
+
+        public override CommandResult ExecuteWithCapturedOutput(string args = "")
```

Comment:
My daily hate of inheritance -- you forget to add override when you change the base class